pages/api/settings.js

```diff
@@ -0,0 +1,16 @@
+export default function handler(req, res){
```

Comment:
Not clear why exactly you make this two api calls and do not even wait for their results here.